sharness/lib/test-lib.sh

```diff
@@ -66,6 +68,9 @@ test_cluster_init() {
         echo "error initializing ipfs cluster"
         exit 1
     fi
+    if [[ -n "$custom_config_files" ]]; then
```

Comment:
Big thing to remember: we use `sh`, not `bash`. Bash features are not available (`[[`). This is why travis build fails.

(I'm not sure why we don't use `/bin/bash` everywhere either, it might be ok but Wyatt used `/bin/sh`. I like that `sh` prevents from doing a lot of magic, as shell scripts + magic is a bad combination though)